src/__test__/server.test.js

```diff
@@ -0,0 +1,166 @@
+'use strict';
```

Comment:
Great test module! Best to rename this as 'country.test.js' or 'country-router.test.js'.